cmd/clusterctl/client/cluster/mover.go

```diff
@@ -275,7 +275,9 @@ func getMoveSequence(graph *objectGraph) *moveSequence {
 		// NB. it is necessary to filter out nodes not belonging to a cluster because e.g. discovery reads all the secrets,
 		// but only few of them are related to Clusters/Machines etc.
 		moveGroup := moveGroup{}
-		for _, n := range graph.getNodesWithTenants() {
+		nodeList := append(graph.getNodesWithTenants(), graph.getNodesWithLabels()...)
```

Comment:
I see a potential problem in this approach, because if by chance a node gets both tenants and labels it will be duplicated in the list.
So I suggest renaming `getNodesWithTenants` in something like `getMoveNodes` and or the "has label/forceMove" conditions test here

Reply:
I didn't worry to much about it because `if moveSequence.hasNode(n)` gets called just below this which should essentiallde-dup any nodes I believe.

Reply:
Uhmm, you are right
However, I still think that it is cleaner and safer to have a unique list (without repetition) than two lists, which are already concatenated in two different points of the codebase (here and in EnsureNamesapaces)

Reply:
right on. I'll make this change. it's what I had originally anyways!

Reply:
updated in amended commit.